1.0/docs/devguide/registering-elements.md

```diff
@@ -232,56 +308,104 @@ local DOM when the element is constructed.
     }
 
 **Note:** This example uses [Automatic node finding](local-dom.html#node-finding) to
-access a local DOM element. 
+access a local DOM element.
 {: .alert .alert-info }
 
-Within a given tree, `ready` is generally called in _document order_, but you should not
-rely on the ordering of initialization callbacks between sibling elements, or between 
-a host element and its light DOM children.
+Within a given tree, `ready` is generally called in _document order_,
+but you should not rely on the ordering of initialization callbacks between
+sibling elements, or between a host element and its **light DOM** children.
+
+### Initialization order and timing {#initialization-order}
+
+The element's basic initialization order for a given element is:
+
+-   `created` callback.
+-   Local DOM initialized (This means that **local DOM** children are created,
+    their property values are set as specified in the template, and `ready`
+    has been called on them).
+-   `ready` callback.
+-   [`factoryImpl` callback](#custom-constructor).
+-   `attached` callback.
 
+Note that while the life cycle callbacks listed above will be called in the
+described order for any given element, the  **initialization timing between
+elements may vary** depending on many factors, including whether or not the
```

Comment:
Can't you at least rely on the parent being initialized before the descendants in its template?